src/App.jsx

```diff
@@ -1,13 +1,26 @@
 import React from 'react';
-// import { Routes, Route } from "react-router-dom";
 import Hero from './components/Hero/Hero';
 import './App.css';
+import LatestJobs from './components/LatestJob/LatestJobs';
 
+const data = [
+  { position: 'Web Developer', salary: '2000$-3000$', date: '10/10/2022' },
+  { position: 'Web Developer', salary: '2000$-3000$', date: '10/10/2022' },
+  { position: 'Web Developer', salary: '2000$-3000$', date: '10/10/2022' },
+  { position: 'Web Developer', salary: '2000$-3000$', date: '10/10/2022' },
+  { position: 'Web Developer', salary: '2000$-3000$', date: '10/10/2022' },
+];
+const header = {
+  title: 'Latest Jobs',
+  first: 'Position',
+  second: 'Salary',
+  third: 'Date',
+};
```

Comment:
It would be best to make this into an array so that you can map over it instead especially since we want to make this component flexible.